### PR TITLE
M1501: CSS Error Reporting: Final Steps Last Part

### PR DESCRIPTION
**Pull request by GauriGNaik**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/8972)

### CODE REVIEW

**Comment by jdm** [comment]
@GauriGNaik It would be better if you could take a recent revision from Servo's master branch, then run `git cherry-pick 77e8721` and push the result instead since the other PR was merged.

File: components/script/dom/window.rs [hunk 1]
```diff
@@ -1323,6 +1323,9 @@ impl Window {
 
         WindowBinding::Wrap(runtime.cx(), win)
     }
+        pub fn return_devtools_flag(&self) -> bool {
```
Comment (nox) [inline components/script/dom/window.rs]: This function is weirdly named, what is it supposed to do?

**Comment by jdm** [comment]
Great! Only minor changes necessary before this can merge!
-S-awaiting-review +S-needs-code-changes 

---

Reviewed 7 of 7 files at r1.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[components/devtools/lib.rs, line 430 \[r1\]](https://reviewable.io:443/reviews/servo/servo/8972#-K5fus6epAAbmwffDDUh:-K5fus6epAAbmwffDDUi:459211392) ([raw file](https://github.com/servo/servo/blob/e52be0348f25f1b6c76f497514321f60755238c1/components/devtools/lib.rs#L430)):*
nit: indent these lines by four spaces past the `let `.

---

*[components/devtools/lib.rs, line 438 \[r1\]](https://reviewable.io:443/reviews/servo/servo/8972#-K5fuwJBIogFGabZhBJH:-K5fuwJBIogFGabZhBJI:-1828666123) ([raw file](https://github.com/servo/servo/blob/e52be0348f25f1b6c76f497514321f60755238c1/components/devtools/lib.rs#L438)):*
nit: unindent this line to match the `D` on the next line.

---

*[components/devtools_traits/lib.rs, line 90 \[r1\]](https://reviewable.io:443/reviews/servo/servo/8972#-K5fv3IlS6MSI8eENAGv:-K5fv3IlS6MSI8eENAGw:207253398) ([raw file](https://github.com/servo/servo/blob/e52be0348f25f1b6c76f497514321f60755238c1/components/devtools_traits/lib.rs#L90)):*
Let's add `/// Report a CSS parse error for the given pipeline.`

---

*[components/script/dom/window.rs, line 1326 \[r1\]](https://reviewable.io:443/reviews/servo/servo/8972#-K5WzHvvoUB1UCLi7zvq-r1-1326:-K5fvDitmFoQ4dpBkgBi:804836196) ([raw file](https://github.com/servo/servo/blob/e52be0348f25f1b6c76f497514321f60755238c1/components/script/dom/window.rs#L1326)):*
I'd prefer a name like `live_devtools_updates` instead. Also, let's unindent all these lines by 4 spaces.

---

*[components/script/script_task.rs, line 2083 \[r1\]](https://reviewable.io:443/reviews/servo/servo/8972#-K5fv_lNZOURKKHAASiA:-K5fv_lNZOURKKHAASiB:-1449886992) ([raw file](https://github.com/servo/servo/blob/e52be0348f25f1b6c76f497514321f60755238c1/components/script/script_task.rs#L2083)):*
nit: deindent this and the next line by four spaces.

---

*[components/script/script_task.rs, line 2085 \[r1\]](https://reviewable.io:443/reviews/servo/servo/8972#-K5fvYjRBTitTzGP6ksl:-K5fvYjRBTitTzGP6ksm:1181138590) ([raw file](https://github.com/servo/servo/blob/e52be0348f25f1b6c76f497514321f60755238c1/components/script/script_task.rs#L2085)):*
nit: indent by two spaces.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/servo/servo/8972#-:-K5fviOg0tdERB_c-7Vq:519631404)*

**Comment by KiChjang** [comment]
@GauriGNaik This PR still needs a rebase for us to merge.

**Comment by jdm** [comment]
@bors-servo: r+
Thanks @gaurignaik!

**Pull request merged**